RxSwift/Traits/PrimitiveSequence.swift

```diff
@@ -124,7 +124,31 @@ extension PrimitiveSequenceType where TraitType == SingleTrait {
             case .error(let error):
                 observer(.error(error))
             case .completed:
-                rxFatalError("SingleProtocol")
+                rxFatalError("Singles can't emit a completion event")
+            }
+        }
+    }
+
+    /**
+     Subscribes a success handler, and an error handler this Single.
```

Comment:
Maybe it would be easier and consistent to just refer Single, Maybe and Completable as observable sequence or sequences.

They are technically just special cases of observable sequences.

It sounds little weird to me when one says: `Subscribes a success handler, and an error handler this Single.`

`Subscription object used to unsubscribe from the observable sequence.`